### ISSUE TITLE
Investigate using an actual logger rather than `console.log`ing random stuff

### ISSUE THREAD

**Issue by 1vasari** [plainblack/Lacuna-Web-Client#95]

**Comment by 1vasari** [comment]
Something like [this](https://github.com/pimterry/loglevel) might do the trick.

**Comment by icydee** [comment]
Yes, I  like the look of that. It seems to be have the same api as log4perl (not surprising since that is itself based on log4j (java). )

By all means, let's try it.

**Comment by 1vasari** [comment]
This is pretty low in the priority list. I'll try it *sometime* no guarantees as to when...
